server.js

```diff
@@ -14,124 +14,59 @@ const { parse } = require('url')
 const next = require('next')
 
 const { getItemByPath } = require('./src/utils/sidebar')
+const { getRedirect } = require('./src/redirects.js')
 
 const dev = process.env.NODE_ENV !== 'production'
 const app = next({ dev })
 const handle = app.getRequestHandler()
 const port = process.env.PORT || 3000
 
-app.prepare().then(() => {
-  createServer((req, res) => {
-    const parsedUrl = parse(req.url, true)
-    const { pathname, query } = parsedUrl
+// Do not run the server if the module is being required by something else
+if (module.parent == null) {
```

Comment:
What is this condition for @fabiosantoscode ? I don't think server.js is ever required by any other source code file.

Reply:
it's not part of the merged latest version it seems

Reply:
@fabiosantoscode let's remove this `if`, we don't need this anymore I think

Reply:
Correct. @jorgeorpinel this was because I was importing server.js from the tests before, as the redirect logic was contained in it.

Reply:
Got it, thanks.